src/Components/WebAssembly/JSInterop/src/WebAssemblyJSRuntime.cs

```diff
@@ -101,6 +101,7 @@ protected override void SendByteArray(int id, byte[] data)
                     return exception != null
                         ? throw new JSException(exception)
                         : (TResult)(object)new WebAssemblyJSObjectReference(this, id);
+                case JSCallResultType.JSDataReference:
```

Comment:
What's this about? It's not clear why we need this here.

Reply:
We added a new `JSCallResultType.JSDataReference`. This type isn't used in WASM. The `default` case would handle this, but I prefer to be explicit about it.

Reply:
Could you add a code comment saying this (and ideally, *why* it's not used)?